modules/object_in_world.py

```diff
@@ -12,7 +12,7 @@ class ObjectInWorld:
 
     @classmethod
     def create(
-        cls, location_x: float, location_y: float, spherical_variance: float
+        cls, location_x: float, location_y: float, spherical_variance: float, label: int
```

Comment:
Is this necessary? The output of `ClusterEstimationByLabel.run()` is already `dict[Label, ObjectInWorld]` .